examples/jsm/shaders/CopyShader.d.ts

```diff
@@ -2,7 +2,7 @@ import {
   Uniform
 } from '../../../src/Three';
 
-export interface CopyShader {
+export const CopyShader: {
```

Comment:
Don't you need a semicolon at the end of the closing bracket like in `FXAAShader`?

Reply:
Yes, I am lazy, wait a moment.